### ISSUE TITLE
Target framework param is required (does not default properly?)

### ISSUE THREAD

**Issue by Phrohdoh** [nunit/dotnet-new-nunit#11]
This project's README.md (and [Microsoft's documentation](https://github.com/dotnet/docs/blob/e70561a236c479b5bdd89450e870d828e7d2bc10/docs/core/testing/unit-testing-with-nunit.md#creating-the-test-project)!) says that `dotnet new nunit` is all you need to run to generate an NUnit project.

This is not the case currently.

```shell
$ dotnet new nunit

Usage: new [options]

Options:
  -h, --help          Displays help for this command.
  -l, --list          Lists templates containing the specified name. If no name is specified, lists all templates.
  -n, --name          The name for the output being created. If no name is specified, the name of the current directory is used.
  -o, --output        Location to place the generated output.
  -i, --install       Installs a source or a template pack.
  -u, --uninstall     Uninstalls a source or a template pack.
  --nuget-source      Specifies a NuGet source to use during install.
  --type              Filters templates based on available types. Predefined values are "project", "item" or "other".
  --force             Forces content to be generated even if it would change existing files.
  -lang, --language   Filters templates based on language and specifies the language of the template to create.


NUnit 3 Test Project (C#)
Author: Aleksei Kharlov aka halex2005 (codeofclimber.ru)
Description: A project that contains NUnit tests that can run on .NET Core on Windows, Linux and macOS
Options:                                                                                    
  -f|--framework    The target framework for the project.                                   
                        netcoreapp1.0    - Target netcoreapp1.0                             
                        netcoreapp1.1    - Target netcoreapp1.1                             
                        netcoreapp2.0    - Target netcoreapp2.0                             
                        netcoreapp2.1    - Target netcoreapp2.1                             
                        net35            - Target net35                                     
                        net40            - Target net40                                     
                        net45            - Target net45                                     
                        net451           - Target net451                                    
                        net452           - Target net452                                    
                        net46            - Target net46                                     
                        net461           - Target net461                                    
                        net462           - Target net462                                    
                        net47            - Target net47                                     
                        net471           - Target net471                                    
                        net472           - Target net472                                    
                    Default: netcoreapp2.0                                                  

  -p|--enable-pack  Whether or not to enable packaging (via ("dotnet pack") for the project.
                    bool - Optional                                                         
                    Default: false / (*) true                                               

  --no-restore      If specified, skips the automatic restore of the project on create.     
                    bool - Optional                                                         
                    Default: false / (*) true                                               


* Indicates the value used if the switch is provided without a value.

$ ls
. ..
```

Notice there was no `.csproj` or `.cs` files created.

But running the following *does* generate the project and C# file.

```shell
$ dotnet new nunit -f netcoreapp2.0
```

**Comment by halex2005** [comment]
It turns out that regression was introduced with latest version 1.5.0.
One thing has been added that can influence - Item Templates for test fixtures.

As temporary workaround, please, specify `--type project` parameter: `dotnet new nunit --type project`.
Alternatively, you can temporarily install previous version of `NUnit3.DotNewNew.Template` with `dotnet new -i `NUnit3.DotNewNew.Template::1.4.0`.

I'll investigate how I can fix this issue.

**Comment by halex2005** [comment]
I've fixed issue, please, use latest version v1.5.1 from [nuget](https://www.nuget.org/packages/NUnit3.DotNetNew.Template).

**Comment by Phrohdoh** [comment]
Great, thank you! I have confirmed that the fix worked for me. :)
